packages/webpack/src/builder.js

```diff
@@ -37,44 +37,32 @@ export class WebpackBundler {
     }
   }
 
+  getWebpackConfig(name) {
```

Comment:
How about returning `config` object here, then we don't need an extract `bind`

Reply:
@clarkdo Can you please elaborate more about bind?

This function is not async to leverage `Promise.all`. The runner will be specified based on build type [here](https://github.com/nuxt/nuxt.js/blob/82abc2632f8da8343d32128377b654838b963f51/packages/webpack/src/builder.js#L106)

Reply:
@clarkdo Refactor applied. Please review again.

Reply:
@pi0 your change was a surprise, but it was appreciated, thank you.